packages/execution-environments/src/common/endowments/network.ts

```diff
@@ -106,7 +106,13 @@ const createNetwork = () => {
     (callback) => callback(),
   );
 
-  const _fetch: typeof fetch = async (
+  const fetch =
+    typeof globalThis.fetch === 'function'
```

Comment:
Should we use `rootRealmGlobal` instead of `globalThis`?

Reply:
https://github.com/MetaMask/snaps-skunkworks/blob/main/packages/execution-environments/src/common/endowments/crypto.ts#L4-L14